phlex-rails.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
 	end
 	spec.require_paths = ["lib"]
 
-	spec.add_dependency "phlex", "~> 1.10.0"
+	spec.add_dependency "phlex", "~> 1.11.0"
```

Comment:
I think we should do:
```ruby
spec.add_dependency "phlex", "~> 1.10"
```
that way it will be compatible with the next version too (`1.12`)

Reply:
It's not clear to me if the maintainers expect phlex-rails to fully work with the next version of phlex always (assuming it would be 1.12). In the past, this constraint was manually adjusted because phlex-rails needed to be updated before it could work with the next release of phlex.